corehq/apps/translations/app_translations/download.py

```diff
@@ -27,37 +27,70 @@ def get_bulk_app_single_sheet_by_name(app, lang):
         prefix = [get_module_sheet_name(module)]
 
         # Name / menu media row
-        # Leave blanks for case_property, list_or_detail, and label
         rows.append(
-            prefix + ['', '', ''] +
-            get_menu_row([module.name.get(lang)],
-                         [module.icon_by_language(lang)],
-                         [module.audio_by_language(lang)]) +
-            [module.unique_id]
+            prefix +
```

Comment:
Much more readable :)

Reply:
> this is making a longer method look even longer

:thinking: What about moving these out into separate functions? In fact, the "Name / menu media row" is identical for forms and modules, so reusing a function would be DRYer. That would make this method shorter, and the functions might be a bit more readable/self-documenting.